### PR TITLE
Updated table of content formatting on several pages

### PR DESCRIPTION
**Pull request by GBKS**
Via #325.

### CODE REVIEW

File: guide/getting-started/common-user-flows.md [hunk 1]
```diff
@@ -44,8 +44,14 @@ Some applications are like Swiss Army Knives. They provide many different indivi
 
 Let's take a look at some common user needs and workflows that can be created for them.
 
-1. Table of contents
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
```
Comment (johnsBeharry) [inline guide/getting-started/common-user-flows.md:50]: what does this line do since it doesn't get outputted on the page? is it for screen readers?

Comment (GBKS) [reply guide/getting-started/common-user-flows.md:50]: It tells the table of contents script whether to display an ordered or unordered list. `1. Table of contents` would create an ordered one.

**Comment by Bosch-0** [comment]
Would it not make these be ordered lists? 

image

image

Also why is this only applied to these two pages? Seems inconsistent.

**Comment by GBKS** [comment]
> Would it not make these be ordered lists?

What do you mean by that?

> Also why is this only applied to these two pages? Seems inconsistent.

"Context over consistency". Just like the discussion with do's and don't styling, icons, and prototypes, I don't think making things consistent is a goal in itself for page content. If you have specific pages where you think a table of contents is helpful to make it easier for users to work with the content, then please bring them up. For this PR, I only converted existing table of contents to the new formatting.

**Comment by Bosch-0** [comment]
> What do you mean by that?

Using 1. 2. 3. instead of dot points

> "Context over consistency".

Consistency guide wide looks a lot cleaner. The personal finance page which has one of these ordered lists and is part of this PR is quite a small page. Most of the other pages in this chapter are of similar length to personal finance - why does personal finance have a list but not the other pages?

**Comment by GBKS** [comment]
> Consistency guide wide looks a lot cleaner.

Something "looking cleaner" is not always the right design decision. That's what "Context over consistency" is about. [See here](https://usabilitypost.com/2008/08/04/context-over-consistency/).

> why does personal finance have a list but not the other pages?

Because that's what felt appropriate for the page at the time of writing.

**Comment by Bosch-0** [comment]
I don't understand the context here, an ordered list on technology primer makes just as much sense as having one on personal finance.

**Comment by GBKS** [comment]
> For this PR, I only converted existing table of contents to the new formatting.

Other changes can be discussed and implemented elsewhere.

**Review by johnsBeharry — APPROVED** [review]
as the scope of change is limited to the pages with the table of contents currently, I see no reason to hold things up.

@GBKS stating the scope of change in the PR description next time would be good. just from the title alone, it was a bit ambiguous, but I saw the comment on the linked issue referring to the intention of this PR.

@Bosch-0 a follow-up PR can be created to implement the change more widely if it has consensus.

**Review by Bosch-0 — APPROVED** [review]

**Pull request merged**